src/io/ASCIIReader.h

```diff
@@ -9,19 +9,20 @@
  */
 class ASCIIReader : public InputBase {
 public:
-	ASCIIReader();
-	~ASCIIReader() {}
+	ASCIIReader ();
 
-	void readXML(XMLfileUnits& xmlconfig);
+	~ASCIIReader () {}
+
+	void readXML (XMLfileUnits& xmlconfig);
```

Comment:
please also undo these kind of changes.

Reply:
Then please change the style guide to include all formatting rules that you
care about.